7/mingw32/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM fedora:38
```

Comment:
Fedora 39 is due out tomorrow

Reply:
Haha, what timing on my part. Let's wait for the Fedora 39 release, then.